### PR TITLE
Testing for data package

### PR DESCRIPTION
**Pull request by mennoo1996**
Increased line coverage to 97.8% in data package

### CODE REVIEW

File: src/test/java/data/CameraShotTest.java [hunk 1]
```diff
@@ -44,5 +44,10 @@ public void setDescriptionTest() {
         cameraShot.setDescription("a");
         assertEquals("a", cameraShot.getDescription());
     }
+    
+    @Test
+    public void toStringTest() {
+        assertEquals("CameraShot()", cameraShot.toString());
+    }
 
 }
```
Comment (Markvanderuit) [inline src/test/java/data/CameraShotTest.java:53]: Github tells me there's something wrong here. Not sure if there is, but thought I'd mention it.

Comment (mennoo1996) [inline src/test/java/data/CameraShotTest.java:53]: What do you mean? I don't see anything remarkable in GitHub?

**Comment by Markvanderuit** [comment]
No further comments. Looks good. Second member can merge!

File: src/test/java/data/CameraShotTest.java (hunk 1 above)
Comment (Markvanderuit) [inline src/test/java/data/CameraShotTest.java:53]: I mean this: http://puu.sh/oO2AX/8d108c8930.png

**Comment by bart-de-jonge** [comment]
Great work bringing the coverage up

File: src/test/java/data/CameraShotTest.java (hunk 1 above)
Comment (bart-de-jonge) [inline src/test/java/data/CameraShotTest.java:53]: No newline is no error

Comment (Markvanderuit) [inline src/test/java/data/CameraShotTest.java:53]: Probably something we can ignore. Weird though.

**Pull request merged**